CONTRIBUTING.md

```diff
@@ -33,4 +33,35 @@ depending on the types of changes defined by
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
-If the required subsection does not exist yet under **Unreleased**, create it!
\ No newline at end of file
+If the required subsection does not exist yet under **Unreleased**, create it!
+
+## Style
+
+### Capitalization and punctuation
+
+Both line comments and doc comments must be capitalized. Each sentence must end with a period.
+
+```
+// This is a line comment.
+```
+
```

Comment:
Are we sure about this?
I think enforcing the use of period makes sense for rustdoc comments. But it feels excessive for regular code comments.